build/yamls/windows/ovs-containerd/conf/Run-AntreaOVS-Containerd.ps1

```diff
@@ -0,0 +1,33 @@
+$ErrorActionPreference = "Stop"
+$mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+$mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
+$newPath = $env:PATH + ";$mountPath/Windows/System32;$mountPath/openvswitch/usr/bin;$mountPath/openvswitch/usr/sbin"
+[Environment]::SetEnvironmentVariable("PATH", $newPath, "User")
+$env:PATH = [Environment]::GetEnvironmentVariable("PATH", "User")
+$OVS_DB_SCHEMA_PATH = "$mountPath/openvswitch/usr/share/openvswitch/vswitch.ovsschema"
```

Comment:
@Atish-iaf 
this line will ensure host file(C:/openvswitch/usr/share/openvswitch/vswitch.ovsschema) accessible inside container with mount path,  instead we should directly use C:/openvswitch/usr/share/openvswitch/vswitch.ovsschema  file from container image as this file is part of latest ovs image inside container ! Going further these files will not exist on host. On host we need to keep conf.db and log file in volumemount location which is accessible inside container